test/Tester/SerializationTests/SerializationTests.DifferentTypes.cs

```diff
@@ -123,16 +122,18 @@ public static void Register()
             }
         }
 
+#if !NETSTANDARD_TODO
         [Fact, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Serialization")]
         public void SerializationTests_RecursiveSerialization()
         {
             TestTypeA input = new TestTypeA();
             input.Collection = new HashSet<TestTypeA>();
             input.Collection.Add(input);
 
-            TestTypeA output1 = TestingUtils.RoundTripDotNetSerializer(input);
+            TestTypeA output1 = Orleans.TestingHost.Utils.TestingUtils.RoundTripDotNetSerializer(input);
```

Comment:
Does this test really serve a purpose? It only tests .NET Framework, not any part of Orleans.